.github/workflows/_deploy-cdk-ihe-gw.yml

```diff
@@ -0,0 +1,188 @@
+name: Reusable IHE GW Deploy workflow based on CDK
+
+# This workflow checks out both the OSS and Internal repos.
+# Because of that, the each step has to indicate the folder in which it should run at - different
+# from regular workflows that only checkout the repo once and run all steps in the same folder.
+
+on:
+  workflow_call:
+    inputs:
+      build_env:
+        description: "the environment we're building to, either 'staging' or 'production' - only apps that have special build scripts need to set this"
+        type: string
+        required: false
+        default: "production"
+      deploy_env:
+        required: true
+        type: string
+        description: "the environment we're deploying to, either 'staging', 'production', or 'sandbox'"
+      cdk_stack:
+        required: true
+        type: string
+        description: "the name of the IHE CDK stack we're deploying"
+      is-branch-to-staging:
+        required: false
+        type: boolean
+        default: false
+        description: "whether this workflow should checkout internal's develop/master or a feature branch"
+      AWS_REGION:
+        required: true
+        type: string
+      IHE_GW_CONFIG_BUCKET_NAME:
+        required: true
+        type: string
```

Comment:
shouldn't this be `required: false` for sandbox?

Reply:
We don't deploy IHE GW on the sandbox